python/dgl/contrib/dis_kvstore.py

```diff
@@ -338,76 +352,157 @@ def init_data(self, name, server_id, shape, init_type='zero', low=0.0, high=0.0)
             id=tensor_shape,
             data=threshold)
         _send_kv_msg(self._sender, msg, server_id)
+
+    def init_local_data(self, name, data_tensor):
```

Comment:
`data_tensor` is data for the local partition?

Reply:
Yes.